api/HideMenuCommand.md

```diff
@@ -8,11 +8,15 @@
 
 `hide_menu menu_type [enable_automatic_transition]`
 
-Hides either the main menu or the pause menu.
+Hides either the main menu or the pause menu. The enable_automatic_transition
+parameter can be used to specify if Escoria manages the graphical transition
+or not. If set to false, you can perform your own transition as part of the
+room's `setup` event (if you want to change the look of the transition used
```

Comment:
I actually think you do it as part of both when exiting a room and when entering a room, so not just `setup` but also the exiting event in the previous room. @StraToN those correspond to transitioning out and in, no?

Reply:
I based that content off @StraToN's feedback in the Help discord channel today. Happy to make whatever changes to it if I didn't capture the intent of his response correctly.